website/docs/core-concepts/stacks/templates/functions/atmos.Component.mdx

```diff
@@ -281,3 +282,65 @@ The template function `{{ atmos.Component "test" .stack }}` is executed three ti
 After the first execution, Atmos caches the result in memory (all the component sections, including the `outputs`),
 and reuses it in the next two calls to the function. The caching makes the stack processing about three times faster in this
 particular example. In a production environment where many components are used, the speedup can be even more significant.
+
+## Using `atmos.Component` with `static` remote state backend
+
+Atmos supports [brownfield configuration by using the remote state of type `static`](/core-concepts/components/terraform/brownfield/#hacking-remote-state-with-static-backends).
+
+For example:
+
+<File title="stack.yaml">
+```yaml
+components:
+  terraform:
+    # Component `test` is configured with the remote state backend of type `static`
+    test:
+      remote_state_backend_type: static
+      remote_state_backend:
+        static:
+          val1: true
+          val2: "2"
+          val3: 3
+          val4: null
+          val5:
+            - item1
+            - item2
+            - item3
+          val6:
+            i1: 1
+            i2: 2
+            i3: 3
+    test2:
+      vars:
+        test_1: '{{ (atmos.Component "test" .stack).outputs.val1 }}'
+        test_2: '{{ (atmos.Component "test" .stack).outputs.val2 }}'
+        test_3: '{{ (atmos.Component "test" .stack).outputs.val3 }}'
+        test_4: '{{ (atmos.Component "test" .stack).outputs.val4 }}'
+        # Use the `!template` Atmos YAML function to correctly handle the outputs of types map and list
+        test_5: !template '{{ (atmos.Component "test" .stack).outputs.val5 }}'
+        test_6: !template '{{ (atmos.Component "test" .stack).outputs.val6 }}'
```

Comment:
I wish we would give more practical examples. E.g. the reason to use this in a brownfield is for example, to set expected outputs, such as for VPC IDs, subnets, etc.

Reply:
For example (asking ChatGPT)

```yaml
static_backend:
  remote_state_backend_type: static
  remote_state_backend:
    static:
      region: "us-west-2"
      cluster_name: "production-cluster"
      vpc_cidr: "10.0.0.0/16"
      database:
        type: "postgresql"
        version: "12.7"
        storage_gb: 100
      allowed_ips:
        - "192.168.1.0/24"
        - "10.1.0.0/16"
      tags:
        Environment: "production"
        Owner: "infra-team"

eks_cluster:
  vars:
    region: '{{ (atmos.Component "static_backend" .stack).outputs.region }}'
    cluster_name: '{{ (atmos.Component "static_backend" .stack).outputs.cluster_name }}'
    vpc_cidr: '{{ (atmos.Component "static_backend" .stack).outputs.vpc_cidr }}'
    db_type: '{{ (atmos.Component "static_backend" .stack).outputs.database.type }}'
    db_storage: '{{ (atmos.Component "static_backend" .stack).outputs.database.storage_gb }}'
    allowed_ips: !template '{{ (atmos.Component "static_backend" .stack).outputs.allowed_ips }}'
    tags: !template '{{ (atmos.Component "static_backend" .stack).outputs.tags }}'
```